stripe2qbo/api/routers/transaction_router.py

```diff
@@ -14,30 +13,18 @@
 
 @router.get("/")
 async def get_all_transactions(
-    db: Annotated[Session, Depends(get_db)],
```

Comment:
you're still going to want the db dependency. This is what creates the connection to the database